### ISSUE TITLE
Appendix A : Solid Resources

### ISSUE THREAD

**Issue by johanbove** [melvincarvalho/solid-hacks#2]
To be able to plan who is writing which content I would like to create issues for each "coming soon" page.

**Comment by johanbove** [comment]
Glad you appreciate my assistance :-)
How does Gitbook work? Does it mean we can’t edit from a regular text editor anymore ? Not sure what the benefit is over using regular Github pages and some Jekyll coding for the navigation and other included and generated sections.

**Comment by melvincarvalho** [comment]
Yes, I'm just experimenting with it right now.  I can add a collaborator, but the UI can be confusing sometimes.

**Comment by johanbove** [comment]
Ok. Experimentation is fun :-)

How does "solid-hacks" actually compare to the Inrupt Docs created by Ruben etc?
https://solid.inrupt.com/docs/

Iow, what is the goal of the solid hacks project?

**Comment by melvincarvalho** [comment]
It's a bit like the hacks books - tips and tricks

http://shop.oreilly.com/product/0636920026273.do

Ruben's docs are much more professional right now, but more time did go into them.

**Comment by melvincarvalho** [comment]
I mailed you an invite to the solid hacks gitbook -- let's see if this helps!

**Comment by melvincarvalho** [comment]
I've uploaded links to appendix A now

https://solid.gitbook.io/solid-hacks/appendix-a-solid-resources
